sdk/core/Microsoft.Azure.Core.NewtonsoftJson/src/NewtonsoftJsonObjectSerializer.cs

```diff
@@ -42,7 +42,10 @@ public NewtonsoftJsonObjectSerializer() : this(CreateJsonSerializerSettings())
         public static JsonSerializerSettings CreateJsonSerializerSettings()
         {
             var settings = new JsonSerializerSettings();
+
+            // TODO: Update the README when converters are added by default.
```

Comment:
nit: NOTE?

TODO sounds a bit like something that wasn't done.

Reply:
Sure. I kinda wrestled with that myself. I wanted it to "pop".